### DISCUSSION TITLE
Get the last user's password change time

### DISCUSSION THREAD

**Discussion by ashep** [zitadel/zitadel#8048] [Q&A] [answered]
What API can I use to retrieve the last user's password change time?

**Comment by ashep** [comment]
@hifabienne hi! Can anyone help me with this?

**Comment by ashep** [comment]
@hifabienne, can you please tell me whether it is possible at all or not? Thank you.

**Comment by mffap** [comment, marked as answer]
That should work with the [events API ](https://zitadel.com/docs/guides/integrate/zitadel-apis/event-api) with the event "Password changed".

_I removed your other comments. Please don't spam the forum._

**Reply by ashep** [reply]
Thank you for the answer. Hope this message is not a spam one.
